docs/how-to-guides/configure-dicom-server-settings.md

```diff
@@ -49,16 +49,42 @@ If you would like to update your SQL Database tier:
 ![Configure SQL2](../images/configure-sql-2.png)
 1. Click **Apply**.
 
-## Additional Configuration Settings
+## Azure Monitor
 
-**Application Insights**: By default, Application Insights is deployed, but not enabled, when you deploy the Medical Imaging Server for DICOM to Azure. To update this setting, navigate to your App Service:
+[Azure Monitor](https://docs.microsoft.com/azure/azure-monitor/overview) offers a variety of solutions to collect, analyze and act on telemetry, including Application Insights Log Analytics.

```

Comment:
Last time as I talked with Brandon , I learned that although appinsights is not enabled, our application is still using AppInsights for logging -- which is a little bit surprise to me. 
![image](https://user-images.githubusercontent.com/67345894/94598067-434b3a00-0243-11eb-9f9d-3d6256b31fb2.png)


I'm not sure how we want to phrase this to customer, also not sure how much appinsights functionalities we are utilizing, could you check with Brandon who knows this much better to figure this out?